Java/syntax_test_java.java

```diff
@@ -603,7 +779,28 @@ void annotatedArgs(@NonNull final String p1,
 // <- meta.annotation.java meta.annotation.parameters.java punctuation.section.parens.end.java
 @fully.qualified.Annotation
 // <- punctuation.definition.annotation.java
-//^^^^^^^^^^^^^^^^^^^^^^^^^ variable.annotation.java
+//^^^^^^^^^^^^^^^^^^^^^^^^^ meta.annotation.java meta.annotation.identifier.java meta.path.java
+//^^^^ variable.annotation.package.java
```

Comment:
@wbond `variable.annotation` is really strange for Java. All annotations must be interface references.

Ideally I would like to see `variable.annotation.package.java` => `support.type.package.java` and `variable.annotation.java` => `support.class.java`. Having two ways of referencing the same thing feels strange. If a theme wants to target the annotations they can use `meta.annotation.identifier`.

Reply:
See https://github.com/sublimehq/Packages/issues/709 and https://github.com/sublimehq/Packages/issues/737 for context.